src/EnergyPlus/IntegratedHeatPump.cc

```diff
@@ -150,98 +147,17 @@ void SimIHP(EnergyPlusData &state,
     case IHPOperationMode::SpaceClg:
         if (!IsCallbyWH) // process when called from air loop
         {
-            SimVariableSpeedCoils(state,
```

Comment:
This set of changes is removing the data members passed into the child object's method. Clang-format also condensed the calls, making it hard to compare.